integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeImpl.java

```diff
@@ -65,24 +66,44 @@
     return sendMessage(instance, integrationUser, streams, message);
   }
 
+  /**
+   * It receives an array of Stream and sends one by one to each room.
+   * Business rule 1: If I have "n" rooms and one receive OK then returns HTTP Status 200
```

Comment:
This isn't the correct rules.  
1. If I have N rooms and all receive OK then return 200
2. If I have N room and at NOT all receive OK then return 500

Reason: the assumption is that the webhook sender will try again if it gets non-200 response.  It is better for Symphony to post the same message twice than to miss a message.